.github/workflows/build-and-push.yml

```diff
@@ -31,6 +31,18 @@ jobs:
             quayio_token: "QUAY_IMAGE_FEDORA_BUILDER_TOKEN"
             tag: "fedora"
             image_name: "nginx-124"
+          - dockerfile: "1.24/Dockerfile.c8s"
+            registry_namespace: "sclorg"
+            quayio_username: "QUAY_IMAGE_SCLORG_BUILDER_USERNAME"
+            quayio_token: "QUAY_IMAGE_SCLORG_BUILDER_TOKEN"
+            tag: "c8s"
+            image_name: "nginx-124-micro-c8s"
+          - dockerfile: "1.24/Dockerfile.c9s"
+            registry_namespace: "sclorg"
+            quayio_username: "QUAY_IMAGE_SCLORG_BUILDER_USERNAME"
+            quayio_token: "QUAY_IMAGE_SCLORG_BUILDER_TOKEN"
+            tag: "c9s"
+            image_name: "nginx-124-micro-c9s"
```

Comment:
```suggestion
            image_name: "nginx-124-c9s"
```

Reply:
Also I see that 1.24/Dockerfile.c9s has .exclude-file. So this will not work.

Reply:
Fix https://github.com/sclorg/nginx-container/pull/278/commits/67ecede3d2da94df21b4fb2c5b0e5fe0349ecdd7

Reply:
It was delete ;)

Reply:
Peto, please, do not close issues that are not resolved. There is a higher change they would pass the review unfixed then.

Are you sure we want to have build and push enabled for nginx-124-c9s when the .exclude-file is present?
What are the reasons for this?

Reply:
@zmiklank The file is not there. It was deleted. And as this is not UBI image, we usually want to provide a C9S image to quay.io.

Reply:
And finally, everything was solved.